logstash-core-event-java/spec/event_spec.rb

```diff
@@ -135,4 +135,54 @@
       expect(event["tags"]).to eq(["foo"])
     end
   end
+
+  module DummyLogger
```

Comment:
this is really specific to this test and the stage the implementation is in now, i.e.: only the warn method implemented. it should stay here.